lighthouse-core/test/results/sample_v2.json

```diff
@@ -279,11 +279,6 @@
             "description": "Application Cache Error event: Manifest fetch failed (404) http://localhost:10200/dobetterweb/clock.appcache",
             "url": "http://localhost:10200/dobetterweb/dbw_tester.html"
           },
-          {
-            "source": "Runtime.exception",
```

Comment:
do you know what happened here? looks like the `level` is `exception` in the artifacts which might have just been an older version

Reply:
Good catch -- you got that right. Updated the JSON